backend/fms_core/resources/sample.py

```diff
@@ -175,7 +175,23 @@ def import_obj(self, obj, data, dry_run):
         except Exception as e:
             individual_created = False
             individual = None
-            errors["individual"] = ValidationError(e.messages.pop(), code="invalid")
+
+        #Individual field errors
+        if not data["Taxon"]:
```

Comment:
Would there be a way to make it more modular (or even re-usable in other Resources which need more precise errors too)? :)

Reply:
I am thinking about creating a function in _utils.py (GenericResource is also an option) and then call that function to validate specific attributes. That way it will be reusable. What do you think?

Reply:
I would retract my comment. if we are to change the tool we use, it may not be worth it for now to invest considerable time making it uniform across all the resources, so what's there works for me :)

Reply:
Great